.github/workflows/cflite_cron.yml

```diff
@@ -0,0 +1,76 @@
+name: ClusterFuzzLite cron tasks
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main  # Use your actual default branch here.
+  schedule:
+    - cron: '0 13 * * 6'  # At 01:00 PM, only on Saturday
+permissions: read-all
+jobs:
+  BatchFuzzing:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address, undefined, memory] # Override this with the sanitizers you want.
```

Comment:
Can't this file be simplified with a 2 entry matrix? Taking mode and sanitizer as parameters? and including only the 5 job you want e.g. something like:

```
    strategy:
      matrix:
        include:
          - mode: batch
            sanitizer: address
          - mode: batch
            sanitizer: undefined
          - mode: batch
            sanitizer: memory
          - mode: prune
            sanitizer: undefined ???
          - mode: coverage
            sanitizer: coverage
```

Reply:
:+1: 
Seems the default value for `sanitizer` field is `address`, and yes, would be good to factorize :wink:

Reply:
Indeed [address](https://github.com/google/oss-fuzz/blob/a5078ad416cfa781426ac091309c17214781b12f/infra/constants.py#L21) is the default sanitizer, I've factorized hope it's okay now